tests/inverted-index-test.js

```diff
@@ -1,10 +1,180 @@
-
+import supertest from 'supertest';
+import path from 'path';
 import indexObj from '../src/inverted-index';
+import app from '../server';
 
 
 indexObj.createIndex('book1.json', indexObj.readFile('book1.json'));
 indexObj.createIndex('book2.json', indexObj.readFile('book2.json'));
-indexObj.createIndex('book2.json', indexObj.readFile('invalidBook.json'));
+
+const createdIndexForBook1 = {
+  'book1.json': {
```

Comment:
These files for testing should be put into an external file and imported into the test suite.